flocker/node/agents/ebs.py

```diff
@@ -1263,15 +1302,22 @@ def aws_from_configuration(region, zone, access_key_id, secret_access_key,
     :param UUID cluster_id: The unique identifier of the cluster with which to
         associate the resulting object.  It will only manipulate volumes
         belonging to this cluster.
+    :param bool no_validate: If False, do not attempt to validate the region
+        and zone by calling out to AWS. Useful for testing.
```

Comment:
Docstring is out of sync w/ parameter name (`no_validate` vs `validate_region`), please update the docstring.